tools/grind_completion

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
how will this work if dev has more than 1 tree of fairscale? they need to source/unsource different completion script when they enter/exit a tree?

We also assume user has "." in her PATH so that grind can be found as ./grind?

Reply:
I'm assuming that `./grind` is always run as `./grind`, from the repo root, not as a PATH command

The hook could be made smarter / multi-path aware.